### PR TITLE
README additions & Feature flag spelling correction

### PR DESCRIPTION
**Pull request by philomates**
- Spelling corrections in some feature flag text
- README notes I added during setting up my HQ dev environment.

Also, both @dcluna and I had to set COUCH_PASSWORD = '' in localsettings.py to get ./manage.py syncdb to run. Maybe something in the README or localsettings.example.py needs to be updated?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -246,12 +246,15 @@ If your installation didn't set up the helper processes required by CommCare HQ
...
     /path/to/unzipped/elasticsearch/bin/elasticsearch &
+
     /path/to/couchdb/bin/couchdb &
 
 Then run the following separately:
 
-    # MacOS Asynchronous task scheduler
+    # Setting up the asynchronous task scheduler
+    # For Mac / Linux
     ./manage.py celeryd --verbosity=2 --beat --statedb=celery.db --events
```
Comment (phillipm) [inline README.md]: I think this works for Linux too. When I ran it, many errors were thrown, but half the manage.py commands I ran threw errors I was then told to ignore by Cory

File: corehq/apps/builds/README.md [hunk 2]
```diff
@@ -1,4 +1,40 @@
-Adding CommCare Builds to CommCare HQ
+Syncing local HQ instance with an Android Phone
```
Comment (phillipm) [inline corehq/apps/builds/README.md]: Moved android instructions to the top, since nobody uses (or should use) J2ME anymore

**Comment by benrudolph** [comment]
love it. good eyes. (can no longer merge)

**Comment by benrudolph** [comment]
also yea, i don't remember how i got to change the `COUCH_PASSWORD`. probably would be good to update docs

**Comment by czue** [comment]
:+1: @phillipm can you resolve conflicts by merging or rebasing with master?

**Comment by phillipm** [comment]
@benrudolph @czue Should be good to go now.

**Comment by czue** [comment]
uh-oh - @phillipm just passed @ctsims on the all time HQ contributors list! https://github.com/dimagi/commcare-hq/graphs/contributors

**Comment by dannyroberts** [comment]
@phillipm feel free to update localsettings.py.example!

**Pull request merged**